package.json

```diff
@@ -136,6 +136,8 @@
     "eslint-plugin-react": "^7.2.1",
     "eslint-plugin-react-hooks": "^1.6.0",
     "jest": "^25.1.0",
+    "jest-circus": "^25.3.0",
```

Comment:
I believe jest-circus ships as the default test runner in jest 25+ so this may not be needed.